### PR TITLE
Nerfs the shit out of the felinid tail grab mood buff.

### PR DESCRIPTION
**Pull request by Iamgoofball**
Mood controls your movespeed. Making Felinids objectively the best mood management race provided your ~~metagame buddy~~friend pulls your tail once every two minutes is insane, even as a meme.

A +5 mood buff was ridiculously good. This is better than the antag mood buff which is 4, equal to the cult buff for sacrificing which is 5, better than tripping balls, better than playing an arcade game and winning, better than the upgraded hug, equal with the best hug, and frankly one of the easiest best mood buffs you can get. And stacks with all the other ones.

:cl:
balance: Nerfs the felinid tail grab mood buff.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Good idea to get this PR in early before it becomes entrenched and people make a big storm of complaints over tail-grabbing being nerfed later (because we all know how this sort of thing goes)

**Comment by redsunoverparadise** [comment]
1984

**Comment by GoldenAlpharex** [comment]
I just want to take a moment to say that in 120 PRs Goof will reach his special `patch-690` branch. Keep it up my guy. Almost there!

File: code/datums/mood_events/generic_positive_events.dm [hunk 1]
```diff
@@ -26,8 +26,9 @@
 
 /datum/mood_event/tailpulled
 	description = "<span class='nicegreen'>I love getting my tail pulled!</span>\n"
-	mood_change = 5
+	mood_change = 1
```
Comment (ATH1909) [inline code/datums/mood_events/generic_positive_events.dm:29]: countersuggestion: make this 2, but make it not stack with the hug and warm hug moodlets

Comment (Iamgoofball) [reply code/datums/mood_events/generic_positive_events.dm:29]: Is that doable with mood code?

**Comment by KathyRyals** [comment]
Yes good, sorry I didn't realize how good it was!

**Comment by RaveRadbury** [comment]
The tail grab applies to lizards, too

File: code/datums/mood_events/generic_positive_events.dm (hunk 1 above)
Comment (ATH1909) [reply code/datums/mood_events/generic_positive_events.dm:29]: yes, it's what warm hugs and hugs do already, IIRC

Comment (ATH1909) [reply code/datums/mood_events/generic_positive_events.dm:29]: wait, maybe not, you'd have to test it

**Comment by Farquaar** [comment]
> 
> Good idea to get this PR in early before it becomes entrenched and people make a big storm of complaints over tail-grabbing being nerfed later (because we all know how this sort of thing goes)

Ah yes. Felinids like it when you target their groin to "pull" their "tail". This is a feature that we can expect to stand the test of time.

**Comment by Zytolg** [comment]
I'd rather take a +1 and have it stack with hugs. Aliens stay winning.

**Comment by 81Denton** [comment]
this shouldn't even be a mood buff in the first place

**Comment by NanoCats** [comment]
Why is it positive at all? I would assume having somebody pull on your tail, if you had one, would be painful if anything????

I play a felinid and even I don't think this should be in the game at all.

**Comment by Stalkeros2** [comment]
Fetishes.

**Comment by GreenTea98** [comment]
> Yes good, sorry I didn't realize how good it was!

then why are you adding shit if you don't know what you're doing :(

**Pull request merged**